NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx

```diff
@@ -122,15 +122,15 @@
   </data>
   <data name="JavaScriptSnippet" xml:space="preserve">
     <value>&lt;script type="text/javascript"&gt;
-	
-    var appInsights=window.appInsights||function(a){{
-        function b(a){{c[a]=function(){{var b=arguments;c.queue.push(function(){{c[a].apply(c,b)}})}}}}var c={{config:a}},d=document,e=window;setTimeout(function(){{var b=d.createElement("script");b.src=a.url||"https://az416426.vo.msecnd.net/scripts/a/ai.0.js",d.getElementsByTagName("script")[0].parentNode.appendChild(b)}});try{{c.cookie=d.cookie}}catch(a){{}}c.queue=[];for(var f=["Event","Exception","Metric","PageView","Trace","Dependency"];f.length;)b("track"+f.pop());if(b("setAuthenticatedUserContext"),b("clearAuthenticatedUserContext"),b("startTrackEvent"),b("stopTrackEvent"),b("startTrackPage"),b("stopTrackPage"),b("flush"),!a.disableExceptionTracking){{f="onerror",b("_"+f);var g=e[f];e[f]=function(a,b,d,e,h){{var i=g&amp;&amp;g(a,b,d,e,h);return!0!==i&amp;&amp;c["_"+f](a,b,d,e,h),i}}}}return c
-    }}({{
-        instrumentationKey: '{0}'
-    }});
 
-    window.appInsights=appInsights,appInsights.queue&amp;&amp;0===appInsights.queue.length&amp;&amp;appInsights.trackPageView();
-	{1}
-    &lt;/script&gt;</value>
+var sdkInstance="appInsightsSDK";window[sdkInstance]="appInsights";var aiName=window[sdkInstance],aisdk=window[aiName]||function(e){function n(e){t[e]=function(){var n=arguments;t.queue.push(function(){t[e].apply(t,n)})}}var t={config:e};t.initialize=!0;var i=document,a=window;setTimeout(function(){var n=i.createElement("script");n.src=e.url||"https://az416426.vo.msecnd.net/scripts/b/ai.2.min.js",i.getElementsByTagName("script")[0].parentNode.appendChild(n)});try{t.cookie=i.cookie}catch(e){}t.queue=[],t.version=2;for(var r=["Event","PageView","Exception","Trace","DependencyData","Metric","PageViewPerformance"];r.length;)n("track"+r.pop());n("startTrackPage"),n("stopTrackPage");var s="Track"+r[0];if(n("start"+s),n("stop"+s),n("addTelemetryInitializer"),n("setAuthenticatedUserContext"),n("clearAuthenticatedUserContext"),n("flush"),!(!0===e.disableExceptionTracking||e.extensionConfig&amp;&amp;e.extensionConfig.ApplicationInsightsAnalytics&amp;&amp;!0===e.extensionConfig.ApplicationInsightsAnalytics.disableExceptionTracking)){n("_"+(r="onerror"));var o=a[r];a[r]=function(e,n,i,a,s){var c=o&amp;&amp;o(e,n,i,a,s);return!0!==c&amp;&amp;t["_"+r]({message:e,url:n,lineNumber:i,columnNumber:a,error:s}),c},e.autoExceptionInstrumented=!0}return t}(
```

Comment:
https://az416426.vo.msecnd.net -- is this constant for all clouds? or separate cdn for non public azure?

Reply:
It would be a different CDN. I wonder if this is something that JS SDK connection string should support?

Reply:
This snippet need to point to a different CDN, then the CDN value should come from connectionstring. @TimothyMothra Can you check if this can be incorporated to connectionstring? Without this, I don't think javascript support can be enabled in non public regions.

Reply:
I just spoke to @markwolff , a CDN for private regions does not exist yet. So We will proceed with this change as is and will have to add support at a later date. I'll discuss this with Reiley so he's aware.